_includes/releases/cloud/2023-04-10.md

```diff
@@ -0,0 +1,25 @@
+## April 10, 2023
+
+<h3> General changes </h3>
+
+- Contract customers can now [create {{ site.data.products.dedicated }} advanced clusters](../cockroachcloud/create-your-cluster.html#step-1-start-the-cluster-creation-process), which have all the features of {{ site.data.products.dedicated }} standard clusters, plus security features needed for [PCI compliance](../cockroachcloud/pci-dss.html). To upgrade your organization to a contract, [contact us](https://cockroachlabs.com/contact-sales).
+- {{ site.data.products.dedicated }} [third party metrics integrations](../{{site.versions["stable"]}}/third-party-monitoring-tools.html) (Datadog and Amazon CloudWatch) will now [export the following metrics](../cockroachcloud/export-metrics.html) to third party targets:
```

Comment:
This is a substantial edit and I may be off base, but I'm working on other docs related to metrics export right now and it seems like this might be the intent? I've updated all links to Dedicated pages rather than the self-hosted docs since these are the Cloud release notes. WDYT? Feel free to edit my edit. :)
```suggestion
- {{ site.data.products.dedicated }} clusters now [export the following metrics](../cockroachcloud/export-metrics.html#the-metricexport-endpoint) the following metrics to third-party monitoring tools such as [Datadog](../cockroachcloud/export-metrics.html?filters=datadog-metrics-export) and [Amazon CloudWatch](../cockroachcloud/export-metrics.html?filters=aws-metrics-export).:
```